src/components/Menu/index.tsx

```diff
@@ -42,7 +45,7 @@ const Menu = (props) => {
       langs={languageList}
       setLang={setLanguage}
       cakePriceUsd={cakePriceUsd.toNumber()}
-      links={config(t)}
+      links={config(t, menuItemsStatus)}
```

Comment:
I think we could turn config into a hook to avoid re-rendering

Reply:
Done